packages/website/src/components/linter/createParser.ts

```diff
@@ -85,6 +85,17 @@ export function createParser(
         typeChecker: checker,
       });
 
+      const compiledFiles =
+        compilerHost.languageService.getEmitOutput(filePath);
+
+      const fileSep = '// --------------------------';
+      compilerHost.createFile(
+        '/compiled',
+        compiledFiles.outputFiles
+          .map(file => `${fileSep}\n// ${file.name}\n${fileSep}\n` + file.text)
+          .join(`\n`),
+      );
```

Comment:
i'm not sure if this is a best place to trigger build

Reply:
🤷 I don't take issue with it, personally.